dist/scss/variables.scss

```diff
@@ -111,6 +111,6 @@ $navigationHoverColor: #191D22;
 // Overlay
 // -------------------------
 $overlayContainerWidthSmall: 580px;
-$overlayContainerWidthMedium: 700px;
+$overlayContainerWidthMedium: 860px;
```

Comment:
But then we also have to adapt [this line](https://github.com/massiveart/husky/blob/develop/scss/modules/overlay.scss#L41), don't we? And maybe some more, but that's the only one I found so far :)